FreeRTOS/iot-reference-esp32c3
# Aligns the Getting Started Guide with IDF v5

# Updating the Getting Started Guide

Description
-----------
Updates the instructions in the getting started guide to reflect [ESP-IDF v5](https://www.espressif.com/en/news/ESP-IDFv5). In particular, the instructions for flashing the certificate partition have changed. Additionally, the `sdk_config` has been updated to preserve backward compatibility.

Test Steps
-----------
Successfully executed the new instructions locally. 

Checklist:
----------

Related Issue
-----------
None

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
